mautic/mautic
# TPROD-356 Removing sortable panels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [y] (added a note to the change log)
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes https://mautic.atlassian.net/browse/TPROD-356



#### Description:

There are PHP templates that we need to refactor to Twig but we couldn't find any place where they are used. The code was introduced in this massive commit https://github.com/mautic/mautic/commit/c90f59d5b56608190fac791f6c008f40db29c733# but I cannot find it used there either. So instead of migrating those PHP templates to Twig we are removing it together with related JS and CSS.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. As it was added in the PR that was adding the ChannelBundle aka Marketing Messages then click through the UI and make sure it works as before.
